persistence-modules/spring-data-jpa-repo-2/src/test/java/com/baeldung/spring/data/persistence/findbyvsfindallby/FindByVsFindAllByIntegrationTest.java

```diff
@@ -0,0 +1,46 @@
+package com.baeldung.spring.data.persistence.findbyvsfindallby;
+
+import com.baeldung.spring.data.persistence.findbyvsfindallby.model.Player;
+import com.baeldung.spring.data.persistence.findbyvsfindallby.repository.PlayerRepository;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = FindByVsFindAllByApplication.class, args = "--spring.jpa.show-sql=true")
```

Comment:
Does this work?

Would it be clearer to just do it this way?

```suggestion
@SpringBootTest(classes = FindByVsFindAllByApplication.class, properties = "spring.jpa.show-sql=true")
```

Reply:
The `args` works too. But, the `properties` looks better. 

Changed!